flask_rebar/rebar.py

```diff
@@ -859,9 +859,17 @@ def _create_json_error_response(
           Additional headers to attach to the response.
         :rtype: flask.Response
         """
-        body = {"message": message}
+        if isinstance(message, messages.ErrorMessage):
+            message_text = message.message
+            rebar_error_code = message.rebar_error_code
+        else:
+            message_text = message
+            rebar_error_code = None
+        body = {"message": message_text}
         if additional_data:
             body.update(additional_data)
+        if rebar_error_code:
+            body["rebar_error_code"] = rebar_error_code
```

Comment:
Perhaps we should set this value as an attribute of `Rebar`? e.g.

```python
class Rebar:
    def __init__(self, ...):
        self.error_code_attr = "rebar_error_code"

    ...

        body[self.error_code_attr] = rebar_error_code
```

It's a little clunky, but it gives an escape hatch for anyone who might not want to expose framework details in their API.

Reply:
Just to make sure I'm tracking correctly, let me try and re-state how I interpret this would work..  
For the "escape hatch," idea is they could explicitly set (given `rebar`, and instance of `Rebar`) `rebar.error_code_attr = None` and if we detect that where we would set `body[self.error_code_attr]` we just forego including that code in the response. 
Does that sound like what you had in mind, @barakalon ?

Reply:
Ah, I was more imagining...

```python
rebar.error_code_attr = "my_application_error_code"
# or...
rebar.error_code_attr = "error_code"
```

Ah, I hadn't thought about the case for omitting the error code entirely.

Reply:
I think we can do both pretty easily 👍

Reply:
Confirmed: easy peasy :) https://github.com/plangrid/flask-rebar/pull/245/commits/d45e1e716ee597149bdf514b9720a5274d0f2e7c

Reply:
Sweet!